client/src/components/landingpage.js

```diff
@@ -0,0 +1,60 @@
+// landingpage
+
+import React, {useState, useEffect} from 'react';
+import axios from 'axios';
+
+function LandingPage() {
+
+	const [icon, chooseIcon] = useState('Anything');
+	const [input, updateInput] = useState("");
+	const [placeholder, setPlaceholder] = useState("Enter a City and State or a Zipcode")
+
+	useEffect(() => {
+		if (navigator.geolocation) {
+			navigator.geolocation.getCurrentPosition(displayLocationInfo)
+		}
+	})
+
+	function displayLocationInfo(position) {
+		const lng = position.coords.longitude;
+		const lat = position.coords.latitude;
+		const API_KEY = `${process.env.REACT_APP_GEO_CODER_API_KEY}`;
+		axios.get(`https://api.opencagedata.com/geocode/v1/json?q=${lat}+${lng}&key=${API_KEY}`)
+			.then((response) => {
+				let city = response.data.results[0].components.town
+				let state = response.data.results[0].components.state_code
+				setPlaceholder(`${city}, ${state}`)
```

Comment:
I'm not familiar with this api - What happens here if we get no results back?  If it still returns a 200, this will throw an error.

Reply:
currently I have it set so that if an error happens it just displays the default placeholder. Should we add an error saying their location could not be found?

Reply:
Yeah i dont think that we need to let them know that the geolocation part didnt find them.  Just a message if the actual search fails to return anything. Which I assume we'll handle in redux or the apollo-graphql tools.